tests/e2e/users/create-meta-db.wdio-spec.js

```diff
@@ -0,0 +1,43 @@
+const _ = require('lodash');
+const utils = require('../../utils');
+const usersPage = require('../../page-objects/users/users.wdio.page');
+const commonElements = require('../../page-objects/common/common.wdio.page');
+const loginPage = require('../../page-objects/login/login.wdio.page');
+const addUserModal = require('../../page-objects/users/add-user-modal.wdio.page');
+
+const username = 'fulltester';
+const fullName = 'Roger Milla';
+const password = 'StrongP@ssword1';
+
+const options = {
+  auth: { username: username, password },
+  method: 'GET',
+  userName: username
+};
+
+describe('Create user meta db : ', () => {
+
+  before(async () => await loginPage.cookieLogin());
+  after(async () => await utils.deleteUsers([{ username: username }], true));
+
+  it('should allow a new user to read/write from meta db', async () => {
+    await usersPage.openAddUserModal();
+    await addUserModal.fillForm(username, fullName, password);
+    await addUserModal.submit();
+    await browser.url(utils.getBaseUrl() + 'messages');
```

Comment:
I think we already have a `commonElements.goToMessages` method available: https://github.com/medic/cht-core/blob/master/tests/page-objects/common/common.wdio.page.js#L91